### PR TITLE
Engines doc

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#145 
#### What does this implement/fix? Explain your changes.
- `Engines` section added to `README.md`
- References section of `README.md` updated
- Test system modified

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,78 @@ Note that it may run on lower end equipment though good performance is not guara
...
+		<td align="center">Max Weight</td>
+		<td align="center">The maximum weight of the edges (if weighted)</td>
+	</tr>
+	<tr>
+		<td align="center">Unsigned / Signed</td>
+		<td align="center">Specifies whether the edge weights are unsigned or signed</td>
+	</tr>
+	<tr>
+		<td align="center">Undirected / Directed</td>
+		<td align="center">Specifies whether the graph is undirected or directed</td>
```
Comment (sadrasabouri) [inline README.md:136]: `signed or unsigned` reads better I guess.

File: README.md [hunk 2]
```diff
@@ -94,6 +94,78 @@ Note that it may run on lower end equipment though good performance is not guara
...
+	</tr>
+	<tr>
+		<td align="center">No Self Loop / Self Loop</td>
+		<td align="center">Specifies whether self-loops are not allowed or allowed</td>
```
Comment (sadrasabouri) [inline README.md:140]: how about `wether self-loop is allowed.`?

File: README.md [hunk 3]
```diff
@@ -94,6 +94,78 @@ Note that it may run on lower end equipment though good performance is not guara
...
+		<td align="center">Vertices Number</td>
+		<td align="center">The total number of vertices in the graph</td>
+	</tr>
+	<tr>
+		<td align="center">Probability</td>
+		<td align="center">The probability for edge creation between any two vertices</td>
+	</tr>
+	<tr>
+		<td align="center">Undirected / Directed</td>
+		<td align="center">Specifies whether the graph is undirected or directed</td>
```
Comment (sadrasabouri) [inline README.md:165]: `signed or unsigned` reads better I guess.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Tiny changes in documents that I guess may be useful.

File: README.md (hunk 3 above)
Comment (sepandhaghighi) [reply README.md]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/169/commits/f5726fca469eb32e946353402771534e5c4ab656

File: README.md (hunk 1 above)
Comment (sepandhaghighi) [reply README.md]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/169/commits/f5726fca469eb32e946353402771534e5c4ab656

File: README.md (hunk 2 above)
Comment (sepandhaghighi) [reply README.md]: fixed in https://github.com/sepandhaghighi/pyrgg/pull/169/commits/70562bd6151c7afd029f1769076efbd397df781b

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**